docs/current/guides/541047-alternative-runtimes.md

```diff
@@ -13,15 +13,39 @@ date: "2023-04-28"
 
 This guide explains how to use Dagger with various OCI-compatible Docker alternatives.
 
-## Podman
+## Approaches
 
-### Requirements
+It is possible to run the Dagger Engine container with any other OCI-compatible container runtime, not just those which are CLI-compatible with Docker. There are two possible approaches.
+
+### Use a `docker` symbolic link
+
+By default, Dagger tries to invoke the `docker` executable. To use a different container runtime instead, create a symbolic link to it in your system path and name it `docker`. This approach is suitable for runtimes which are CLI-compatible with Docker.
+
+### Run the Dagger Engine container manually
+
+An alternative approach is to run the Dagger Engine container manually and set the `_EXPERIMENTAL_DAGGER_RUNNER_HOST` environment variable to point to the running container. If this variable is set, then Dagger will instead connect to the endpoint specified there. This approach is suitable for runtimes which are not CLI-compatible with Docker, or when using a customized Dagger Engine container.
+
+The `_EXPERIMENTAL_DAGGER_RUNNER_HOST` variable currently accepts values in the following format:
+
+| Format | Description |
+|------- | ------------|
+| `docker-container://<container-name>` | Connect to the runner inside the given Docker container. Requires the `docker` CLI to be present and usable. Will result in shelling out to `docker exec`. |
+| `podman-container://<container-name>` | Connect to the runner inside the given Podman container. |
+| `kube-pod://<pod-name>?context=<context>&namespace=<namespace>&container=<container>` | Connect to the runner inside the given Kubernetes pod. Query strings params like `context` and `namespace` are optional.|
```

Comment:
Do we have a guide `dagger with k8s` That could be useful because I'm not sure I understand how I can use it with dagger? Do my dagger pod need to be inside the k8s cluster? Or just the engine and then I can use my cli from my localhost? I'm curious

Reply:
Sure, please ping me in Discord and I can provide more information.